Makefile

```diff
@@ -6,4 +6,14 @@ changelog:
 release:
 	semtag final -s minor
 
+.PHONY: test
 test:
+	pre-commit run
+
+.PHONY: test-slow
```

Comment:
Remove "slow." Naming should identify the behavior, not describe it quantitatively. `test-single-region` and `test-all-regions` would be an appropriate alternative.

Reply:
i think names have a lot of purposes, not just for identifying behavior.

i am not sure i want more concrete names like test-single-region if the implementation of the tests will likely change over time. The only structure I've put in place is for deciding where to place a test case based on how slow it is.

Down the road, the tests may be stable enough that we can start determining which tests people are expected to
run before submitting a change and monitor to ensure that those tests are fast enough.

Reply:
I am not persuaded. Names are an identifying label. Right now you have two test modes, single region and all region. If the tests change their intended behavior, the names should change too.

"Slow" is not an identifier for the declared tests, it is a descriptor of an emergent behavioral property, i.e. a side effect of how the tests are defined. 

In a huge test suite, you might have something like `--skip-slow` or `--skip-integration` etc, because the number of tests is so large that a caller could not realistically be expected to identify and skip them by name. In a suite of 2 tests, this is a premature abstraction.

Reply:
sounds good, i can give concrete names here